### PR TITLE
New package: hunspell-sv_SE-2.41

### PR DESCRIPTION
**Pull request by uruba**

### CODE REVIEW

File: srcpkgs/hunspell-sv_SE/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'hunspell-sv_SE'
+pkgname=hunspell-sv_SE
+version=2.41
+revision=1
+noarch=yes
+create_wrksrc=yes
+hostmakedepends="unzip"
+short_desc="Swedish sv_SE dictionary for hunspell"
+maintainer="Venca Uruba <uruba@outlook.com>"
+license="GNU-GPL-v3, GNU-GPL-v2, LGPL-v3+"
```
Comment (maxice8) [inline srcpkgs/hunspell-sv_SE/template]: Please use the spdx short-identifier for the licenses.

LGPL-v3+ is LGPL-3.0-or-later

Please check whether GNU-GPL-v3 correspond to GPL-3.0-or-later or GPL-3.0-only
Please check whether GNU-GPL-v2 correspond to GPL-2.0-or-later or GPL-2.0-only

File: srcpkgs/hunspell-sv_SE/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+homepage="https://extensions.libreoffice.org/extensions/swedish-spelling-dictionary-den-stora-svenska-ordlistan/"
+distfiles="https://EXtensions.libreoffice.org/extensions/swedish-spelling-dictionary-den-stora-svenska-ordlistan/$version/@@download/file/ooo_swedish_dict_$version.oxt>dict-da-$version.zip"
+checksum=48c8d0d548232215384c47ba18ac03bbef3af4c11ffd5e1057272353ba60225f
+
+do_install() {
+	vinstall dictionaries/sv_SE.aff 644 usr/share/hunspell
+	vinstall dictionaries/sv_SE.dic 644 usr/share/hunspell
+	vinstall dictionaries/sv_FI.aff 644 usr/share/hunspell
+	vinstall dictionaries/sv_FI.dic 644 usr/share/hunspell
+	vlicense LICENSE_en_US.txt
```
Comment (maxice8) [inline srcpkgs/hunspell-sv_SE/template]: GNU GPL licenses are in the system under /usr/share/licenses, you do not need to use vlicense to install them, vlicense is used for licenses like MIT and ISC that have copyright headers that need to be included with the package.

File: srcpkgs/hunspell-sv_SE/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	vlicense LICENSE_sv_SE.txt
+}
+
```
Comment (maxice8) [inline srcpkgs/hunspell-sv_SE/template]: Extra line at the end, please remove

File: srcpkgs/hunspell-sv_SE/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'hunspell-sv_SE'
+pkgname=hunspell-sv_SE
+version=2.41
+revision=1
+noarch=yes
+create_wrksrc=yes
+hostmakedepends="unzip"
+short_desc="Swedish sv_SE dictionary for hunspell"
+maintainer="Venca Uruba <uruba@outlook.com>"
+license="LGPL-3.0-or-later"
```
Comment (maxice8) [inline srcpkgs/hunspell-sv_SE/template]: GPL-2.0-or-later, GPL-3.0-or-later, LGPL-3.0-or-later

**Comment by uruba** [comment]
Alright, sorry for this, I should have paid closer attention to the manual. I have elected to specify just the "LGPL-3.0-or-later" license, since that is what corresponds to the included license file (what had been there before had basically just been blindly copied from the extension's website).

**Comment by maxice8** [comment]
On Wed, Feb 13, 2019 at 03:29:43AM -0800, Václav Uruba wrote:
> Alright, sorry for this, I should have paid closer attention to the manual. I have elected to specify just the "LGPL-3.0-or-later" license, since that is what corresponds to the included license file (what had been there before had basically just been blindly copied from the extension's website).
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8630#issuecomment-463164846

Oh, ok, please disregard the other comment then.

**Pull request merged**